app/views/verify/address/index.html.slim

```diff
@@ -1,5 +1,14 @@
-p Pick an address verification mechanism
+h1.h2
+  = t('idv.titles.select_verification')
+p
+  = t('idv.messages.select_verification', sp_name: @sp_name)
```

Comment:
For now, I think the easiest route would be to show different copy depending on whether an sp is present or not.  Check out `DecoratedSession` and how we are using that for rendering different content for sp and non-sp sessions. (basically, you define the same method on `ServiceProviderSessionDecorator` and `SessionDecorator` and have them return different content)

let me know if you'd like to pair on this! should be pretty fast and then we can get this merged :) :) 🌈